File: src/main/java/no/nav/domain/pensjon/regler/beregning2011/GReguleringDetaljer.java
package no.nav.domain.pensjon.regler.beregning2011;

import java.io.Serializable;
import java.util.Date;

public class GReguleringDetaljer implements Serializable {

    private Date justeringTomDato;
    private double justeringsfaktor;
    private int forrigeG;
    private int gjeldendeG;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/GReguleringDetaljer.kt
package no.nav.pensjon.regler.domain.beregning2011

import java.io.Serializable
import java.util.*

@Deprecated("Slettes. Ikke produsert av pensjon-regler.")
class GReguleringDetaljer : Serializable, IGRegulering {
    override var justeringTomDato: Date? = null
    override var justeringsfaktor = 0.0
    override var forrigeG = 0
    override var gjeldendeG = 0
}